graphql/schemas/Ecosystem/users.graphql

```diff
@@ -4,13 +4,42 @@ type User {
     firstname: String!
     lastname: String!
     displayname: String!
+    description: String
     email: String!
-    phone: String
+    phone_number: String
+    cell_phone_number: String
+    address_1: String
+    address_2: String
+    zip_code: Int
+    sex: String
     user_active: Boolean!
     created_at: DateTime
     updated_at: DateTime
     branches: [CompanyBranch!] @HasManyThrough
     companies: [Company!] @HasManyThrough
+    city: City @belongsTo
+    state: State @belongsTo
+    country: Country @belongsTo
+}
+
+type City {
```

Comment:
@arfenis we already have them defined you can call them directly here https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/graphql/schemas/Ecosystem/countries.graphql#L1